app/preprints/-components/preprint-institutions/institution-select-list/template.hbs

```diff
@@ -1,30 +1,25 @@
-{{#paginated-list/has-many
-    local-class='institutionList'
-    model=@manager.user
-    relationshipName='institutions'
-    analyticsScope='User Institutions'
-    usePlaceholders=this.usePlaceholders
-    as |list|
-}}
-    <list.item as |institution|>
-        {{#if institution}}
-            <label data-test-institution={{institution.id}} local-class='label'>
-                {{#let (includes institution @manager.affiliatedList) as |isAffiliated|}}
+{{#if this.displayComponent}}
+    <div local-class='institution-list-container {{if (is-mobile) 'mobile'}}'>
+        <label data-test-affilitated-institutions-label>
+            {{t 'preprints.submit.step-metadata.institutions.label'}}
+        </label>
+        <p>
+            {{this.descriptionDisplay}}
+        </p>
+        {{#each @manager.institutions as |institution index|}}
+            <div local-class='institution-container'>
+                <div local-class='institution-checkbox'>
                     <Input
-                        data-test-institution-button={{concat (if isAffiliated 'remove' 'add') '-' institution.id}}
-                        data-analytics-name={{if isAffiliated 'Remove' 'Add'}}
-                        disabled={{@manager.shouldDisable}}
+                        data-test-institution-input={{index}}
                         @type='checkbox'
-                        @checked={{isAffiliated}}
+                        @checked={{institution.isSelected}}
                         {{on 'change' (fn @manager.toggleInstitution institution)}}
                     />
-                {{/let}}
-                {{institution.name}}
-            </label>
-        {{else}}
-            <ContentPlaceholders as |placeholder|>
-                {{placeholder.list items=2}}
-            </ContentPlaceholders>
-        {{/if}}
-    </list.item>
-{{/paginated-list/has-many}}
+                </div>
+                <div local-class='label' data-test-institution-name={{index}}>
+                    {{institution.name}}
+                </div>
+            </div>
```

Comment:
Missed this earlier, but would it be possible to add a label to this input for accessibility like
```
<label>
    <Input />
    {{institution.name}}
</label>
```
or
```
{{#let (unique-id) as |id|}}
    <label for={{some-id}}>
        {{institution.name}}
    </label>
    <Input />
{{/let}}
```
`institution.name` can be either before or after the input element, of course